docs/install.rst

```diff
@@ -50,6 +53,24 @@ tested using the `GNU-RM toolchain
     link time optimization is enabled during the MicroPython build
     (LTO is enabled by default).
 
+Building via Docker
+~~~~~~~~~~~~~~~~~~~
+
+To build via docker, simply invoke `docker/shell` from the root directory after
+having docker installed. This will build the docker image and also drop you
+into a shell with wasp os's source code shared into the container at
+``/home/user/wasp-os``. All ``make`` commands should be usable from this shell,
+including ``make sim`` and ``make check``.
+
+.. note::
+
+    If you want to use the Docker-based setup, it is assumed that you're using
```

Comment:
I think this also needs to mention that docker does not support the BLE tools (wasptool, ota-dfu, etc) so not all functionality is available via docker.